.gitignore

```diff
@@ -9,3 +9,4 @@ heroku
 *.log
 docs/_site/
 tmp/
+.git/*
```

Comment:
Adding this to `.gitignore` mostly because VSCode will search through all the old history when looking for matching text. It's super annoying and we don't need to be committing anything in `.git` anyway.